src/components/Error.jsx

```diff
@@ -1,15 +1,14 @@
 import { useHistory } from 'react-router-dom';
 import errorPhoto from '../../public/404.jpg';
+import * as S from './Theme/GlobalStyles';
```

Comment:
I don't like naming like S. It's not obvious

Reply:
it may or may not be a convention on a project. But could be easily changed to Styled, right @virzhinia-petkova ? :)

Reply:
I already changed it to exactly Styled :)